webapp/channels/src/components/admin_console/license_settings/license_settings.test.tsx

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {describe, beforeAll, afterAll, expect} from '@jest/globals';
```

Comment:
Why are we having to import `describe` and such explicitly in some of these test files? I'd prefer if we didn't need that, but I'm also surprised we only need to do it in a few places for some reason

Reply:
yes removed we can tackle this upgrade later